### PR TITLE
fix: fixed some small bugs

### PR DESCRIPTION
**Pull request by sg-s**
## changes

see inline comments

### CODE REVIEW

File: src/data_hub/api.py [hunk 1]
```diff
@@ -343,6 +334,9 @@ def assign_files_to_cell(
         },
     ]
 
+    if row_id:
+        rows[0]["rowId"] = row_id
+
```
Comment (sg-s) [inline src/data_hub/api.py:339]: @kennyjwilli suggested this which is simpler

File: src/data_hub/api.py [hunk 2]
```diff
@@ -505,7 +499,7 @@ def set_data_in_cells(
     column = [
         col
         for col in response.cols
-        if col["id"] == column_id or col["name"] == column_id or col["key"] == column_id
```
Comment (sg-s) [inline src/data_hub/api.py:508]: no more column keys

File: src/data_hub/api.py [hunk 3]
```diff
@@ -568,7 +562,7 @@ def set_cell_data(
...
```
Comment (sg-s) [inline src/data_hub/api.py:571]: no more column keys

**Review by kennyjwilli — APPROVED** [review]
Nice simplification 👍🏻

**Pull request merged**